.gitlab-ci.yml

```
@@ -1,14 +1,16 @@
 
 variables:
   DOCKER_REPOSITORY: mendersoftware/mender-convert
+  MULTIPLATFORM_BUILD: "true"
+  MULTIPLATFORM_PLATFORMS: "linux/amd64,linux/arm64"
   S3_BUCKET_NAME: mender-convert-images
 
   # These variables are present elsewhere in the repository too. Make sure to
   # search for and change them too.
   MENDER_ARTIFACT_VERSION:
-    value: "master"
+    value: "3.11.2"
     description: |-
-      Version of mender-artifact for the builder image - 'master' or a specific version
+      Version of mender-artifact for the builder image - an specific version
```

Not my strongest field, but it looks like a typo ;) 
```suggestion
      Version of mender-artifact for the builder image - a specific version
```